source/extensions/http/cache/file_system_http_cache/file_system_http_cache.cc

```diff
@@ -388,6 +404,93 @@ void FileSystemHttpCache::trackFileRemoved(uint64_t file_size) {
   stats_.size_bytes_.set(size_bytes_);
 }
 
+FileSystemHttpCache::EvictionsRequired FileSystemHttpCache::needsEviction() const {
+  EvictionsRequired required;
+  if (config().has_max_cache_size_bytes()) {
+    // capture a value from the atomic because we're going to use it twice and don't want our
+    // value to change in between.
+    uint64_t seen_size = size_bytes_;
+    required.size_bytes_ = seen_size > config().max_cache_size_bytes().value()
```

Comment:
make a temp for max_cache_size_bytes(); same for the count below.

Reply:
Done, but then undone because instead we don't do this any more.